### PR TITLE
Try to generalize advanced vertical tabs usage

### PR DESCRIPTION
**Pull request by chrfritsch**
I tried to generalize the usage of the advanced vertical tabs a bit.

With this changes, everything would be fine for Thunder for Print as well

### CODE REVIEW

File: thunder_admin.theme [hunk 1]
```diff
@@ -200,9 +200,9 @@ function thunder_admin_theme_suggestions_form_element_alter(&$suggestions, $vari
 }
 
 /**
- * Implements hook_form_alter().
+ * Implements hook_form_FORM_ID_alter().
```
Comment (dbosen) [inline thunder_admin.theme]: I know, that phpcs complains about this, but this is not an implementation of hook_form_FORM_ID_alter, so it should not be stated.

Comment (chrfritsch) [inline thunder_admin.theme]: So, what should i write to satisfy phpcs?

File: thunder_admin.theme [hunk 2]
```diff
@@ -206,6 +200,15 @@ function thunder_admin_theme_suggestions_form_element_alter(&$suggestions, $vari
 }
 
 /**
+ * Implements hook_form_FORM_ID_alter().
+ */
+function thunder_admin_form_alter(&$form, FormStateInterface $form_state, $form_id) {
+  if (!empty($form['advanced']) && (in_array('node_edit_form', $form['#theme']) || in_array('node_form', $form['#theme']))) {
```
Comment (chrfritsch) [inline thunder_admin.theme]: Maybe we should remove the in_array's, but then e.g. media will look like this. But the in_array's couples it to the node form. 

bildschirmfoto 2017-07-26 um 15 29 04

File: thunder_admin.theme (hunk 1 above)
Comment (mtodor) [inline thunder_admin.theme]: Try something like this:
`Implements hook_form_alter() for thunder_admin theme.`

**Pull request merged**